sql/enumtype.go

```diff
@@ -87,11 +93,17 @@ func CreateEnumType(values []string, collation CollationID) (EnumType, error) {
 		}
 		/// The elements listed in the column specification are assigned index numbers, beginning with 1.
 		valToIndex[value] = i + 1
+
+		byteLength := uint32(len([]rune(value)) * int(maxCharLength))
```

Comment:
Rather than `[]rune(value)`, we could use `utf8.RuneCountInString(value)` which won't allocate a new slice. Go may optimize this check to this function anyway, but I think it's slightly more readable as well.

Reply:
oh sweet! thanks for showing me that method.